stdlib/public/Darwin/Accelerate/Quadrature.swift

```diff
@@ -46,6 +46,7 @@
 ///         }
 ///     }
 ///
+///     let diameter = 25.0
 ///     let vRresult = quadrature.integrate(over: 0.0 ... diameter,
```

Comment:
I think this was meant to match the example above, so just stating `0.0 ... 25.0` is fine.

Reply:
Fixed